### DISCUSSION TITLE
Welcome to LinkNavigator Discussions!

### DISCUSSION THREAD

**Discussion by interactord** [interactord/LinkNavigator#27] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by insub4067** [comment]
Great

**Comment by vikram-kumar-wits** [comment]
How can we hide tabBar for some screen? Like pushing a view in fullscreen.

**Reply by interactord** [reply]
I'll update the sample code for you.
